### DISCUSSION TITLE
Some ideas from discord

### DISCUSSION THREAD

**Discussion by ChanceSD** [ChanceSD/PvPManager#543] [Ideas]
Putting them here so I don't forget, if anyone has more feel free to share your opinions too.

---
> A block on drinking potions/removing a specific effect or a total block on the server would suffice. Browsing the internet, I have not found anything for newer versions that enables something like this

---

> Specifically, it would be incredibly useful if PvP Manager could offer the ability to:
> 
> 1️⃣ Disable specific commands if a player has PvP enabled or disabled. For example, preventing certain actions (like joining a duel or interacting with PvP-related commands) unless PvP is in a specific state (enabled or disabled).
> 2️⃣ Only allow a commands to be used when PvP is enabled or disabled, providing more control over the behavior of commands based on PvP status.
> 
> I understand the plugin already offers a solid API, but I was wondering if this type of feature could be considered for future updates. Let me know if this is something that could be integrated or if I've missed a way to implement this already. Thanks for considering my suggestion!

**Comment by PerpetuallyLost** [comment]
Hey, Chance! Thanks for considering my suggestion :)

**Reply by ChanceSD** [reply]
No problem, this is for a future more complete feature. But if all you need is mostly to block that duel issue then we can probably add a simple solution in the meantime.

**Reply by PerpetuallyLost** [reply]
Yeah, that'd be great - it's just I don't want people with disabled PvP in a duel or the ability to disable PvP while IN a duel
